src/lib/compile-nools-rules.js

```diff
@@ -1,112 +1,57 @@
+const path = require('path');
+
 const fs = require('./sync-fs');
-const jshintWithReport = require('./jshint-with-report');
-const minifyJs = require('./minify-js');
+const package = require('./package-lib');
```

Comment:
`package` is a reserved keyword. Maybe use `pack` instead, like in `compile-contact-summary` ?

Reply:
Done